bot/persistence/database_connector.py

```diff
@@ -309,9 +364,7 @@ def get_suggestion(self, suggestion_id: int) -> Optional[tuple]:
             result = db_manager.execute(queries.GET_SUGGESTION_BY_ID, (suggestion_id,))
 
             row = result.fetchone()
-            if row:
-                return row
-            return None
+            return row
```

Comment:
No none check? What happens on a failed insertion, eg. problem with the db?

Reply:
This is just a method which gets data regarding a specific suggestion. If ther isn't a suggestion with the specified ID, `None` will be returned. The bot handles this.